### PR TITLE
use e2e f.Timeouts.PodDelete instead of 1 minute

### LINKED ISSUE #108891: [flake][sig-node] Pods should run through the lifecycle of Pods and PodStatus [Conformance]
### Which jobs are flaking?

pull-kubernetes-conformance-kind-ga-only-parallel

### Which tests are flaking?

[sig-node] Pods should run through the lifecycle of Pods and PodStatus [Conformance] 

### Since when has it been flaking?

After https://github.com/kubernetes/kubernetes/pull/93459 is merged.

### Testgrid link

https://testgrid.k8s.io/sig-testing-kind#pull-kubernetes-conformance-kind-ga-only-parallel&include-filter-by-regex=should%20run%20through%20the%20lifecycle%20of%20Pods%20and%20PodStatus

### Reason for failure (if possible)

It relies on events which are not reliable as discussed [here](https://github.com/kubernetes/kubernetes/issues/71646).

### Anything else we need to know?

_No response_

### Relevant SIG(s)

/sig node

### PR DESCRIPTION
**Pull request by pacoxu**
#### What type of PR is this?
/kind flake

#### Which issue(s) this PR fixes:

Fixes #108891

#### Does this PR introduce a user-facing change?

```release-note
None
```

### CODE REVIEW

File: test/e2e/common/node/pods.go [hunk 1]
```diff
@@ -1042,7 +1042,7 @@ var _ = SIGDescribe("Pods", func() {
 		framework.ExpectNoError(err, "failed to delete Pod by collection")
 
 		ginkgo.By("watching for the Pod to be deleted")
-		ctxUntil, cancel = context.WithTimeout(ctx, 1*time.Minute)
```
Comment (pacoxu) [inline test/e2e/common/node/pods.go:1045]: @Riaankl  @dashpole Do you have the context that why we use 1 minute for a pod deletion here?

https://github.com/kubernetes/kubernetes/blob/dc3f5ec6ccb9855dfa99f4c1078625df5fdfab6a/test/e2e/framework/timeouts.go#L26-L27

We meet some flake in https://testgrid.k8s.io/sig-release-master-informing#periodic-kubernetes-e2e-kind-kms https://github.com/kubernetes/kubernetes/issues/108891, so I change it to 5 minute which is the default Pod deletion timeout.

Comment (aojea) [reply test/e2e/common/node/pods.go:1045]: it seems is an aleatory number https://github.com/kubernetes/kubernetes/commit/079764ebd5477c77acb11a55587d2499f9c25138 , we should use the common timeouts for consistency

/lgtm
/approve

**Comment by aojea** [comment]
it seems is an aleatory number https://github.com/kubernetes/kubernetes/commit/079764ebd5477c77acb11a55587d2499f9c25138 , we should use the common timeouts for consistency

/lgtm
/approve

**Pull request merged**